### DISCUSSION TITLE
Notifications not just for mentions

### DISCUSSION THREAD

**Discussion by noraa-junker** [community/community#3422] [Mobile]
I want a ability to costumize the notifications you get on your phone.

Currently you can only choose if you want push notifications for mentions or not. But I want a ability where I can choose that I want to get all my notifications as push notifications.

**Comment by PranamBhat** [comment]
Can you please elaborate the feedback/suggestion?

**Reply by noraa-junker** [reply]
> Can you please elaborate the feedback/suggestion?

Done

**Reply by PranamBhat** [reply]
@Aaron-Junker I can give you these options :

1) Go to "**Settings**" in your mobile and **Turn ON** all notifications for "**GitHub**" app.

2) Enable "All Notifications" in your "GitHub" app.

3) You can go for third-party apps like : [https://github.com/manosim/gitify-mobile](https://github.com/manosim/gitify-mobile)

For more information : [https://docs.github.com/en/github/managing-subscriptions-and-notifications-on-github/configuring-notifications](https://docs.github.com/en/github/managing-subscriptions-and-notifications-on-github/configuring-notifications)
